dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -177,3 +181,41 @@
     "POSTGRES_VERSION": "--postgres-version",
     "MSSQL_VERSION": "--mssql-version",
 }
+
```

Comment:
I tihnk we can move those to a separate file. `global_constants` is not a good place. 

maybe `path_utils.py` ? or maybe even we should have "utils" module and move it to "utils/path_utils.py" ?

Then possibly current "utils.py" could be moved to "utils/run_utils.py"

Reply:
Just make sure `utils/__init__.py` is there too :)